internal/provider/resource_project_test.go

```diff
@@ -11,6 +11,12 @@ import (
 	"testing"
 )
 
+// The domain to add to the project. The TLD should be authorized in your test Vercel account.
+const domainAlias = "domain-alias.chronark.com"
+
+// The repository to test project creation with. Your Vercel account should have access to this.
+const repository = "chronark/terraform-provider-vercel"
```

Comment:
@chronark I added these constants to make it easier to run it against my own account. Ideally these could be over-ridden by env-vars, but that's probably better done in a separate PR.

Reply:
Yes good idea. I have created #40 to keep it in mind